example/provider-extensions/gardener-discovery-server/deploy.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+#
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+usage() {
+  echo "Usage:"
+  echo "> deploy.sh [ -h | <garden_kubeconfig> <seed_kubeconfig> ]"
+  echo
+  echo ">> For example: deploy.sh ~/.kube/garden-kubeconfig.yaml ~/.kube/garden-kubeconfig.yaml"
```

Comment:
```suggestion
  echo ">> For example: ${0} ~/.kube/garden-kubeconfig.yaml ~/.kube/garden-kubeconfig.yaml"
```

When the script is used via other name (file is copied or linked), the usage will report the actual name of the file used to call the script.